tests/test_integration_layout.py

```diff
@@ -70,7 +69,7 @@
             # #root's virtual height should be as high as its stacked content
             (SCREEN_W - 2 - 1, PLACEHOLDERS_DEFAULT_H * 4),
             # placeholders width=same than screen, minus 2 borders, minus scrollbar :: height=default height minus 2 borders
-            (SCREEN_W - 2 - 1, PLACEHOLDERS_DEFAULT_H),
+            (SCREEN_W - 2 - SCROLL_V_SIZE, PLACEHOLDERS_DEFAULT_H),
```

Comment:
this test was assuming that scrollbar sizes was always `1` by default. Now that it's no longer the case we'd better use constants to store the default scrollbar sizes of our Widgets :slightly_smiling_face: